### DISCUSSION TITLE
Disable Blank Name

### DISCUSSION THREAD

**Discussion by JaminShanti** [jamulussoftware/jamulus#2941] [General]
Someone is logging in to an excessive number of Jamulus servers with blank name at the same time.
Can I request Blank name is not allowed.  Maybe don't allow the same IP to login to 20 plus Jamulus servers at the same time too.

**Comment by rdica** [comment]
I like the idea of the client not permitting sending a blank username on connect, but the latter is not a realistic expectation.
Servers are independently owned and operated, they don't have any connection to other servers thus they cannot know what any other server is doing. Nor should they. As a server operator, I don't want some upstream determining who/what can connect to my servers, or how many clients from the same IP I allow.

+1 on the blank username.

**Comment by JaminShanti** [comment]
Someone is abusing the app and logging in excessively with a blank username. Suggestions on what to do about it? It is essentially breaking the pound option when filtering users.

**Reply by rdica** [reply]
https://github.com/jamulussoftware/jamulus/discussions/2935

**Comment by ann0see** [comment]
@JaminShanti we are aware of this issue. Please block this IP if you own a server and comment on #2935 

Locking as discussion should be in #2935
